randovania/games/dread/logic_database/Dairon.txt

```diff
@@ -544,7 +544,7 @@ Extra - asset_id: collision_camera_008
   > Door to Wide Beam Room
       After Dairon - Power Switch 1
   > Tunnel to Power Switch 1
-      Can Slide
+      Trivial
```

Comment:
This should probably have requirements to break the blocks

Reply:
It looks like there's a major inconsistency throughout the DB regarding shot blocks. Some are just trival, some list the templates. Speaking of which the template is just trivial, which I assume is a placeholder for later.